src/main/java/seedu/cakecollate/model/order/UniqueOrderList.java

```diff
@@ -134,4 +135,9 @@ private boolean ordersAreUnique(List<Order> orders) {
         }
         return true;
     }
+
```

Comment:
While it may already be intuitive, maybe you can add a comment that the function sorts the dates according to the earliest to the latest.

Reply:
Good point, added :)